pyomo/contrib/doe/measurements.py

```diff
@@ -26,15 +26,15 @@
 #  ___________________________________________________________________________
 
 
-from pyomo.common.dependencies import (
-    numpy as np, numpy_available,
-    pandas as pd, pandas_available,
-    matplotlib as plt, matplotlib_available,
-)
+#from pyomo.common.dependencies import (
```

Comment:
Why was this commented out?

Reply:
We are not using these dependencies in `measurement`. Removed this sentence